ukwa/npld-player
# Setup UI library and JS renderer

(https://github.com/ukwa/npld-player/issues/5) Sets up [Shoelace](https://shoelace.style/) UI library with Webpack for bundling the renderer script, and renders app bar. UI is not interactive yet.

### Manual testing
Run app in dev with `yarn start`. Verify that app starts up and shows following app bar at top.

### Screenshots
After closing dev tools:
<img width="912" alt="Screen Shot 2022-04-30 at 10 37 05 PM" src="https://user-images.githubusercontent.com/4672952/166133728-4035ce52-a274-42db-92d6-17e68bf8f547.png">
